streaming/Streaming.mli

```diff
@@ -1018,6 +1025,10 @@ module Stream : sig
   val filter : ('a -> bool) -> 'a t -> 'a t
   (** A stream that includes only the elements that satisfy a predicate. *)
 
+  val filter_map : ('a -> 'b option) -> 'a t -> 'b t
+  (** Filter_map f source applies f to every element x of source, discard it if
+   * f x is None, and keeps y if f x = Some y *)
```

Comment:
Same here, removed leading `*` on the second line and make `Filter_map` lowercase`.